lua/metals.lua

```diff
@@ -63,6 +63,47 @@ M.sources_scan = function()
   })
 end
 
+M["metals/quickPick"] = function(_, _, resp)
+  local ids = {}
+  local labels = {}
+  for i, item in pairs(resp.items) do
+    table.insert(ids, item.id)
+    table.insert(labels, i .. '- ' .. item.label )
```

Comment:
```suggestion
    table.insert(labels, i .. ' - ' .. item.label )
```

Nitpick, can we add a space here as well. I think it looks a bit better this way.